commands/Utilities/guild.js

```diff
@@ -17,21 +17,61 @@ module.exports = class extends Command {
     }
 
     async run(msg) {
+      const guild = msg.guild
+      const server = {
+        name: guild.name,
+        id: guild.id,
+        createdAt: guild.createdAt,
+        image: guild.iconURL(),
+        botJoin: guild.joinedAt,
+        region: guild.region.toUpperCase(),
+        owner: `${guild.owner.displayName}#${guild.owner.user.discriminator}`,
+        filter: guild.explicitContentFilter,
+        roles: guild.roles.array().length,
+        members: guild.memberCount,
+        bots: guild.members.filter((member) => { return member.user.bot }).array().length,
```

Comment:
`guild.members.filter(member => member.user.bot).size`
this wont work correctly anyway, not all members are cached.